### DISCUSSION TITLE
USR W610 Setup problems

### DISCUSSION THREAD

**Discussion by Parisn0** [StealthChesnut/HA-FoxESS-Modbus#32] [Q&A] [answered]
After following the USR W610 Setup Guide, all the sensors in Home Assistant are either 0 or Unavailable, Upon running modpoll

C:\WINDOWS\system32>D:\Downloads\modpoll-3.10\win\modpoll.exe -m tcp -p 502 -a 247 -t 3 -0 -1 -c 16 -r 11000 192.168.10.209
modpoll 3.10 - FieldTalk(tm) Modbus(R) Master Simulator
Copyright (c) 2002-2021 proconX Pty Ltd
Visit https://www.modbusdriver.com for Modbus libraries and tools.

Protocol configuration: MODBUS/TCP, FC4
Slave configuration...: address = 247, start reference = 11000 (PDU), count = 16
Communication.........: 192.168.10.209, port 502, t/o 1.00 s, poll rate 1000 ms
Data type.............: 16-bit register, input register table

-- Polling slave...
Invalid frame error!

If I change the slave address to 1 (-a 1) 

C:\WINDOWS\system32>D:\Downloads\modpoll-3.10\win\modpoll.exe -m tcp -p 502 -a 1 -t 3 -0 -1 -c 16 -r 11000 192.168.10.209
modpoll 3.10 - FieldTalk(tm) Modbus(R) Master Simulator
Copyright (c) 2002-2021 proconX Pty Ltd
Visit https://www.modbusdriver.com for Modbus libraries and tools.

Protocol configuration: MODBUS/TCP, FC4
Slave configuration...: address = 1, start reference = 11000 (PDU), count = 16
Communication.........: 192.168.10.209, port 502, t/o 1.00 s, poll rate 1000 ms
Data type.............: 16-bit register, input register table

-- Polling slave...
Invalid reply error!

Any ideas?

**Comment by Parisn0** [comment, marked as answer]
After spending most of the day on this it seems I had a faulty connection to the USR W610.

NOTE configuration.yaml contains an error in that there is an extra space on line 120

**Reply by Parisn0** [reply]
Thanks to all involved with this, awesome piece of work

**Comment by williamjeccles** [comment]
I've corrected the indenting in this commit, thanks for the feedback https://github.com/StealthChesnut/HA-FoxESS-Modbus/commit/c5429c5c1f306fb1006739fa59ec53400f03d9d5
